docs/en/advanced/i18n.md

```diff
@@ -50,39 +50,95 @@ To change the locale, use the `setLocale` method.
 
 ## Configuration
 
-The framework supplies a default configuration automatically, inside `I18nBootloader`. To alter the configuration,
-create the `app/config/translator.php` file:
+The framework supplies a default configuration automatically, inside `Spiral\Bootloader\I18nBootloader`. If you need
+custom settings, you can define them in `app/config/translator.php`:
+
+### Configuration File
+
+Here's what a typical configuration might look like:
 
 ```php app/config/translator.php
 use Symfony\Component\Translation\Dumper;
 use Symfony\Component\Translation\Loader;
 
 return [
-    'locale'         => env('LOCALE', 'en'),
+    'locale' => env('LOCALE', 'en'),
     'fallbackLocale' => env('LOCALE', 'en'),
-    'directory'      => directory('locale'),
-    'autoRegister'   => env('DEBUG', true),
+    'directory' => directory('locale'),
+    'autoRegister' => env('DEBUG', true),
     // available locale loaders (the key is extension)
-    'loaders'        => [
+    'loaders' => [
         'php'  => Loader\PhpFileLoader::class,
         'po'   => Loader\PoFileLoader::class,
         'csv'  => Loader\CsvFileLoader::class,
         'json' => Loader\JsonFileLoader::class
     ],
     // export methods
-    'dumpers'        => [
+    'dumpers'  => [
         'php'  => Dumper\PhpFileDumper::class,
         'po'   => Dumper\PoFileDumper::class,
         'csv'  => Dumper\CsvFileDumper::class,
         'json' => Dumper\JsonFileDumper::class,
     ],
-    'domains'        => [
+    'domains' => [
         // by default we can store all messages in one domain
         'messages' => ['*']
     ]
 ];
 ```
 
+**Key Points:**
+
+1. **Locale Settings:** The `locale` is your primary language, while `fallbackLocale` is the default language the app
+   will use if the specified one isn't available.
+2. **Directory Setting:** This is where the language files reside. By default, it's set to the `app/locale` directory.
+3. **Loaders and Dumpers:** These are classes that handle reading from and writing to translation files.
+
+### Changing the Directory for Locale Files
+
+If you want to use a different directory for your language files, modify the `directory` option:
+
+```php app/config/translator.php
+return [
+    'directory' => directory('locale'),
+];
+```
+
+Sometimes, you might need translations from more than one location. For example, when you're using external packages
+that bring their own translations.
+
+To specify multiple directories use `directories` option instead of `directory`:
```

Comment:
Option `directories` are used for additional directories (for example, from packages). There is no need to move directory('locale') into it. It should stay in the `directory` option.